docs/rst/statistics_backend/get_data.rst

```diff
@@ -0,0 +1,118 @@
+.. include:: ../exports/alias.include
+
+.. _statistics_backend_get_data:
+
+Retrieve statistical data
```

Comment:
I may choose a title more similar to the function name, or at least add the function name in the title. i.e. `get_entities` -> `Get entities of a given kind`